index.html

```diff
@@ -0,0 +1,448 @@
+                <a
+                  class="text-sm font-semibold text-secondary hover:text-primary"
+                  href="#"
+                  >News</a
+                >
+              </li>
+              <li>
+                <a
+                  href="#"
+                  class="text-secondary hover:text-primary active:border-2 p-2 active:font-semibold border-primary"
+                >
+                  CC Campaign
+                </a>
+              </li>
+            </ul>
+          </div>
+          <a
+            href="#"
+            id="hamburger-button"
+            class="md:hidden"
+            aria-label="hamburger menu"
+          >
+            <img
+              width="32px"
+              height="32px"
+              src="./assets/icons/hamburger.svg"
+              alt="menu-icon"
+            />
+          </a>
+        </nav>
+      </header>
+      <section
+        id="welcome"
+        class="my-12 p-4 xl:px-32 lg:px-16 md:px-8 flex flex-col justify-center items-start"
+      >
+        <p class="text-xl text-primary my-1 md:text-3xl">
+          "Hello! Sharing world"
+        </p>
+        <div
+          class="bg-[url('./assets/backgrounds/title.jpg')] bg-clip-text text-transparent bg-center bg-cover"
+        >
+          <h1 class="text-5xl font-bold md:text-8xl">
+            HELLO TOMORROW GLOBAL SUMMIT 2023
+          </h1>
+        </div>
+        <div
+          class="p-4 my-8 border-2 text-sm border-white border-solid lg:max-w-[60%]"
+        >
+          <p>
+            Under the high patronage of Mr. Emmanuel Macron president of the
+            French republic, Welcome to the epiccenter of Deep Tech, We present
+            you the new era of technology
+          </p>
+        </div>
+        <h2 class="text-3xl my-2 font-bold">2023.03.9(FRI) ~ 10(SAT)</h2>
+        <p class="text-sm font-semibold mb-8">
+          @ The CENTQUATRE-PARIS, 5 Rue Curial, 75019 Paris France
+        </p>
+      </section>
+    </main>
+    <section
+      id="main-program"
+      class="flex flex-col justify-center items-center gap-2 py-4 xl:px-32 lg:px-16 md:px-8 bg-[url('./assets/backgrounds/programs.jpg')] bg-center"
+    >
+      <h2 class="text-light text-2xl mt-8 font-semibold">Main Program</h2>
+      <div class="w-12 h-[2px] rounded mb-8 bg-primary"></div>
+      <ul
+        id="programs"
+        class="flex flex-col md:flex-row flex-wrap gap-2 justify-center py-4"
+      >
+        <li
+          class="grid grid-cols-2 md:grid-cols-1 bg-[#383b4277] items-center mx-2 pr-2 md:p-4 hover:border-2 border-light cursor-pointer md:h-72 md:w-52 md:mx-0"
+        >
+          <div
+            class="flex justify-start md:justify-center md:flex-col gap-2 items-center p-4"
+          >
+            <img
+              class="w-16 h-auto"
+              src="./assets/icons/lecture.png"
+              alt="lecture-icon"
+            />
+            <p class="text-xl font-bold text-primary w-max">Lecture</p>
```

Comment:
- [ ] Kindly make sure that the text doesn't overlay one another to match the given design.
- [ ] Please remove the white space that appears on the right side of the index page in mobile view.
![image](https://user-images.githubusercontent.com/62290545/217674845-22c4a554-560c-493a-82ab-47fdb7982861.png)

Reply:
Thanks @ShadyShawkat for your kind feedback, I didn't noticed that, I will make the required changes 👨‍💻